packages/mobile/e2e/src/usecases/OnRamps.js

```diff
@@ -117,18 +204,26 @@ export default onRamps = () => {
         await element(by.text('Next')).tap()
       })
 
-      jest.retryTimes(2)
       it('Then Should Display Exchanges & Account Address', async () => {
-        await waitFor(element(by.id('provider-Binance')))
-          .toBeVisible()
-          .withTimeout(30 * 1000)
-        await expect(element(by.id('provider-Binance'))).toBeVisible()
-        await expect(element(by.id('provider-Bittrex'))).toBeVisible()
-        await expect(element(by.id('provider-Coinbase (CELO as CGLD)'))).toBeVisible()
-        await expect(element(by.id('provider-Coinbase Pro (CELO as CGLD)'))).toBeVisible()
-        await expect(element(by.id('provider-CoinList Pro'))).toBeVisible()
-        await expect(element(by.id('provider-OKCoin'))).toBeVisible()
-        await expect(element(by.id('accountBox'))).toBeVisible()
+        await waitForElementId('accountBox')
+        await expect(element(by.id('provider-Binance'))).toExist()
```

Comment:
We were talking in eng sync, maybe one of the idea of preventing the e2e test from failing when the response from the cloud func changed is having less specific assertions. We can assert on there are list of exchange providers instead of specifying which exchange they are. In this case I feel pretty comfortable as long as the cloud func returns some exchanges, the test case we are testing here should be passing